grails-app/controllers/org/pih/warehouse/UrlMappings.groovy

```diff
@@ -439,6 +439,14 @@ class UrlMappings {
             action = [DELETE: "removingItem"]
         }
 
+        // Putaway API
+
+        // MOBILE
+        "/api/listPutaways"(parseRequest: true) {
```

Comment:
If this interferes with the current `api/putaways` from develop, it is fine if it's split, but I would put it under a different endpoint (I'd avoid the `listPutaways` here)